Patches/Better Infestations/Patch_Race_Insect.xml

```diff
@@ -6,7 +6,7 @@
 		<operations>
 		 <!-- Checking for Better Infestations -->
 			<li Class="CombatExtended.PatchOperationFindMod">
```

Comment:
While these patches are being updated, please change the PatchOp from the old CE FindMod to the newer vanilla FindMod. It's better for troubleshooting and removes one potential point of failure if the CE FindMod breaks in the future.

Reply:
Alright, that will be done.